packages/edit-site/src/components/save-panel/index.js

```diff
@@ -19,6 +25,20 @@ import { store as editSiteStore } from '../../store';
 import { unlock } from '../../private-apis';
 import { useActivateTheme } from '../../utils/use-activate-theme';
 
+function isPreviewingTheme() {
+	return (
+		window?.__experimentalEnableThemePreviews &&
+		getQueryArg( window.location.href, 'theme_preview' ) !== undefined
+	);
+}
+
+function currentlyPreviewingTheme() {
```

Comment:
These functions are already defined in `packages/edit-site/src/utils/is-previewing-theme.js` so we can reuse them here.